lib/knock.rb

```diff
@@ -16,8 +16,18 @@ module Knock
   self.token_signature_algorithm = "HS256"
 
   # Configure the key used to sign tokens.
-  mattr_accessor :token_secret_signature_key
-  self.token_secret_signature_key = -> { Rails.application.secrets.secret_key_base }
+  mattr_writer :token_secret_signature_key, default: -> { Rails.application.secrets.secret_key_base }
+
+  class EmptySecretKey < StandardError
```

Comment:
I think this class can be moved to a separated file. Perhaps it would be nice to have all project exception classes inside a folder named `exceptions` or `errors`